### DISCUSSION TITLE
图片加载慢怎么办还有观看人数都是一样的

### DISCUSSION THREAD

**Discussion by Pardoferilis** [share121/inter-knot#1079] [其他]
顺便找找搭子，米氏游戏和瓦英农，重度游戏者
![5db7c79026a2e1316be0990b5250307](https://github.com/user-attachments/assets/86a486aa-82a5-4d63-ae0a-b1120cef93fd)
https://space.bilibili.com/94861147?spm_id_from=333.1007.0.0

**Comment by Pardoferilis** [comment]
哦哦哦

**Comment by Kamisato100** [comment]
之前浏览数量是随机动态的，估计现在更新了代码还没修好

**Comment by VacuolePaoo** [comment]
bug请移步issues反馈
